pkg/env/env.go

```diff
@@ -39,6 +39,9 @@ type Interface interface {
 	ManagedDomain(string) (string, error)
 	MetricsSocketPath() string
 	Zones(vmSize string) ([]string, error)
+	GenerateACRToken() bool
+	ACRResourceGroup() string
+	ACRName() string
```

Comment:
We are going to need to know the subscription ID of the ACR too.  Don't assume it's the same as the RP's subscription ID.  Combine all of this down into ACRResourceID(), make it configurable via ARM template parameter -> environment variable.